dist/css/general.min.css

```diff
@@ -645,6 +645,30 @@ html{font-family:Roboto,sans-serif;font-weight:400;font-size:62.5%;}
 .grid-col-xlg-11{width:91.66667%;}
 .grid-col-xlg-12{width:100%;}
 }
+.link--blue,.link{color:#0091ea;}
+.link--blue:hover,.link:hover{color:#0071b7;}
+.link--blue:active,.link:active{color:#005284;}
+.link--green{color:#7cb342;}
+.link--green:hover{color:#628e34;}
+.link--green:active{color:#486827;}
+.link--yellow{color:#ffb300;}
+.link--yellow:hover{color:#cc8f00;}
+.link--yellow:active{color:#996b00;}
+.link--orange{color:#fb8c00;}
+.link--orange:hover{color:#c87000;}
+.link--orange:active{color:#955300;}
+.link--red{color:#f83c00;}
+.link--red:hover{color:#c53000;}
+.link--red:active{color:#922300;}
+.link--white{color:rgba(255,255,255,0.87);}
+.link--white:hover{color:#fff;}
+.link--white:active{color:#fff;}
+.link--gray{color:gray;}
+.link--gray:hover{color:#666;}
+.link--gray:active{color:#4d4d4d;}
+.link--black{color:#333;}
+.link--black:hover{color:#1a1a1a;}
+.link--black:active{color:#1a1a1a;}
```

Comment:
The default color is blue, without modifier, just `link`, see: https://github.com/jusbrasil/monalisa/pull/25/files#diff-ca1cd9a9fdb371994559a0bc08840131R18